blockly-ws-search/webpack.config.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/**
+ * @fileoverview Webpack configuration file.
+ * @author aschmiedt@google.com (Abby Schmiedt)
+ */
+
+const path = require('path');
+const webpack = require('webpack');
+
+module.exports = env => {
+    return {
+        target: 'web',
+        mode: env.mode,
+        entry: './src/WorkspaceSearch.js',
+        devtool: 'source-map',
+        output: {
+            path: path.resolve(__dirname, 'dist'),
+            filename: 'workspace-search.umd.js',
+            libraryTarget: 'umd',
+            umdNamedDefine: true,
+            globalObject: 'this'
```

Comment:
What does globalObject: this do?

Reply:
https://github.com/webpack/webpack/issues/6522#issuecomment-557451468

Reply:
That makes sense, curious why we're not defining ``library``.

Reply:
Okay great, just checking.